src/index.ts

```diff
@@ -0,0 +1,4 @@
+/* istanbul ignore file */
```

Comment:
Is this needed if the project is using jest?

Reply:
this was me being nit to exclude `index.js` from the coverage report 😅